web/client/components/misc/cardgrids/SideCard.jsx

```diff
@@ -26,30 +26,33 @@ const React = require('react');
  * @prop {function} onClick callback on card click
  */
 
-module.exports = ({body, className = '', style = {}, onClick=() => {}, size, title, preview, description, caption, tools, selected, ...more} = {}) =>
-<div className={`mapstore-side-card${selected ? ' selected' : ''}${size ? ' ms-' + size : ''} ${className}`}
-    onClick={() => onClick({title, preview, description, caption, tools, ...more})}
-    style={style}>
-    <div className="ms-head">
-        {preview && <div className="mapstore-side-preview">
-            {preview}
-        </div>}
-        <div className="mapstore-side-card-info">
-            <div className="mapstore-side-card-title">
-                <span>{title}</span>
+
+module.exports = ({body, className = '', style = {}, onClick=() => {}, size, title, preview, description, caption, tools, selected, ...more} = {}) => {
+    const {onMouseEnter, onMouseLeave} = more;
```

Comment:
To avoid to pass these 2 methods to onClick, destructure props instead of `more`